src/DotNetty.Transport/Channels/Sockets/AbstractSocketByteChannel.cs

```diff
@@ -153,22 +153,8 @@ public override void FinishRead(SocketChannelAsyncOperation operation)
         protected override void ScheduleSocketRead()
         {
             SocketChannelAsyncOperation operation = this.ReadOperation;
-            bool pending;
-#if NETSTANDARD1_3
-            pending = this.Socket.ReceiveAsync(operation);
-#else
-            if (ExecutionContext.IsFlowSuppressed())
```

Comment:
`ExecutionContext` was not available .net standard 1.3. It is in 2.0 so we should keep `#else` branch here (and elsewhere with `ExecutionContext`) instead.